server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/message/view/MessageFastView.java

```diff
@@ -0,0 +1,113 @@
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.draft.model.message.view;
+
+import java.time.Instant;
+import java.util.Optional;
+
+import org.apache.james.jmap.api.preview.Preview;
+import org.apache.james.jmap.draft.methods.JmapResponseWriterImpl;
+import org.apache.james.jmap.draft.model.BlobId;
+import org.apache.james.jmap.draft.model.Emailer;
+import org.apache.james.jmap.draft.model.Keywords;
+import org.apache.james.jmap.draft.model.Number;
+import org.apache.james.jmap.draft.model.PreviewDTO;
+import org.apache.james.mailbox.model.MailboxId;
+import org.apache.james.mailbox.model.MessageId;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * This message view holds all Message properties expected to be fast.
+ */
+@JsonDeserialize(builder = MessageFastView.Builder.class)
+@JsonFilter(JmapResponseWriterImpl.PROPERTIES_FILTER)
+public class MessageFastView extends MessageHeaderView {
+
+    public static MessageFastView.Builder<? extends MessageFastView.Builder> builder() {
+        return new Builder();
+    }
+
+    @JsonPOJOBuilder(withPrefix = "")
+    public static class Builder<S extends MessageFastView.Builder<S>> extends MessageHeaderView.Builder<S> {
+        protected Optional<Preview> preview;
+
+        protected Builder() {
+            super();
+        }
+
+        public S preview(Preview preview) {
+            this.preview = Optional.of(preview);
+            return (S) this;
+        }
+
+        public S preview(Optional<Preview> preview) {
+            this.preview = preview;
+            return (S) this;
+        }
+
+        public MessageFastView build() {
+            checkState();
+            return new MessageFastView(id, blobId, threadId, mailboxIds, Optional.ofNullable(inReplyToMessageId),
+                headers, Optional.ofNullable(from),
+                to.build(), cc.build(), bcc.build(), replyTo.build(), subject, date, size, PreviewDTO.from(preview),
+                keywords.orElse(Keywords.DEFAULT_VALUE));
+        }
+
+        public void checkState() {
```

Comment:
public?

Reply:
it's `@Overrided` I forgot to put the annotation